components/code-buddy.html

```diff
@@ -0,0 +1,12 @@
+<section class="teams margin-t34">
+  <ul class="teams__list">
+    <li class="teams__items code-buddy">
```

Comment:
I would specify more what exactly this class is about. For someone external, it will be hard to follow what a code-buddy is doing.

```suggestion
    <li class="teams__items code-buddy">
    
    code-buddy-container 
    
```
This can also be a utility class, right?

Reply:
I kind of agree with Berenice and already commented on this earlier. I would not make it a utility class but it's own component.